net/knock/Portfile

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem		1.0
+PortGroup		github 1.0
+
+github.setup		jvinet knock 258a27e5a47809f97c2b9f2751a88c2f94aae891
+version			0.7.8-20151227
```

Comment:
You have some spacing issues. Please use spaces instead of tabs and align everything.